pkg/sql/rowenc/index_fetch.go

```diff
@@ -54,6 +57,18 @@ func InitIndexFetchSpec(
 		encoding.EncodedLengthUvarintAscending(uint64(s.TableID)) +
 		encoding.EncodedLengthUvarintAscending(uint64(index.GetID())))
 
+	if ext := table.ExternalRowData(); ext != nil {
+		newCodec := keys.MakeSQLCodec(ext.TenantID)
```

Comment:
Could I get more insight into what MakeSQLCodec does? I understand generally it provides the details for the external table but how?